front-end/src/components/main-page/TranscriptDescription.test.js

```diff
@@ -0,0 +1,16 @@
+import {render, screen, waitFor} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import "@testing-library/jest-dom";
+import TranscriptDescription from "./TranscriptDescription.jsx"
+import {exampleTranscript} from "../helpers/ExampleTranscript.js"; 
+
+describe("TranscriptDescription tests", () => {
```

Comment:
Tests are implied, we should remove these from our `describe` headers.

```suggestion
describe("TranscriptDescription", () => {
```

Reply:
since a lot of be test files don't have tests in it, I think we should just delete the tests in all the fe files. Thanks helia!